rest_framework_reactive/pool.py

```diff
@@ -265,7 +272,7 @@ def stop_all(self):
         Stops all query observers.
         """
 
-        for observer in self._observers.values():
+        for observer in list(self._observers.values()):
```

Comment:
Maybe add a note then. It was not obvious to me that `observer.stop()` would modify the underlying data structure.